### ISSUE TITLE
update plugin for UE 5.5

### ISSUE THREAD

**Issue by jinpavg** [Cycling74/RNBOMetasound#57]

**Comment by CharleyLanusse** [comment]
yes, please! +1

**Comment by jinpavg** [comment]
Hey @x37v , and others --

Indeed, it looks like my issues were on macOS only, and were resolved by updating Xcode and creating a fresh project and a fresh pull of `RNBOMetasound` -- either `main` or `develop` works for me with RNBO 1.3.3.

@CharleyLanusse -- A note that if you are updating your UE project from a previous version of unreal, you'll probably need to clone a fresh version of this RNBOMetasound repo as your plugin. UE seems to have a hard time recompiling the plugin when it has previously been built for another version of UE. 

One more note that for some reason, UE crashed the first time I tried to build my 5.5 project with RNBOMetasound included. But when I tried again, it built successfully. The crash seemed to have to do with metadata specifiers for the UE version on macOS, not the RNBOMetasound plugin itself.
